subworkflows/local/ont/main.nf

```diff
@@ -0,0 +1,41 @@
+include { PREPARE_ONT } from '../prepare_ont/main'
+include { JELLYFISH } from '../jellyfish/main'
+
+workflow ONT {
+    take:
+    input_channel
+
+    main:
+    Channel.empty().set { genome_size }
+    Channel
+        .fromPath("${projectDir}/assets/report/.DUMMYFILE")
+        .map { it -> [null, it] }
```

Comment:
inserting `null`'s in a workflow is generally not advised. Either use empty lists, empty maps, or empty strings where appropriate. But also remember to remove these instances of `.DUMMYFILE` as nf-core has a better way of doing it.